py/server/deephaven/table.py

```diff
@@ -1661,6 +1701,76 @@ def slice(self, start: int, stop: int) -> Table:
         except Exception as e:
             raise DHError(e, "table slice operation failed.") from e
 
+    def rollup(self, aggs: Union[Aggregation, Sequence[Aggregation]], by: Union[str, Sequence[str]] = None,
+               include_constituents: bool = False) -> RollupTable:
+        """Creates a rollup table.
+
+         A rollup table aggregates by the specified columns, and then creates a hierarchical table which re-aggregates
+         using one less by column on each level. The column that is no longer part of the aggregation key is
+         replaced with null on each level.
+
+         Note some aggregations can not be used in creating a rollup tables, these include: group, partition, median,
+         pct, weighted_avg
```

Comment:
What makes each of these aggs not work?  (e.g. median, pct, and weighed_avg)

Reply:
@rcaudy Maybe you can explain?

Reply:
Should we then have a note that is less specific, with the hope that these things do get fixed?  Then we don't have to remember that these notes need to be updated.

Reply:
I'm not sure. This is Python specific messaging; the Java code has its own, right down in the authoritative code.

Reply:
I'm not sure. This is Python specific messaging; the Java code has its own, right down in the authoritative code.

Reply:
I am OK with leaving it as is because the individual failure tests will catch the change if we do get these fixed in the future.